test/e2e/mainnetBootnodes.test.ts

```diff
@@ -25,6 +25,9 @@ describe("discv5 integration test", function () {
       ip6: multiaddr(`/ip6/::/udp/${port++}`),
     },
   ]) {
+    // ip6 test fails in github runner
+    if (process.env.CI && bindAddrs.ip6) return;
```

Comment:
If this test is run in mocha, it's best to do `this.skip()` to signal test existence on all suites